x/data-proxy/keeper/keeper.go

```diff
@@ -79,6 +79,16 @@ func (k Keeper) processProxyFeeUpdate(ctx sdk.Context, pubKeyBytes []byte, proxy
 		UpdateHeight: updateHeight,
 	}
 
```

Comment:
By the way, I think this function should be renamed to `scheduleFeeUpdate()`, right?